br/pkg/lightning/backend/kv/sql2kv.go

```diff
@@ -267,7 +272,7 @@ func logKVConvertFailed(logger log.Logger, row []types.Datum, j int, colInfo *mo
 		log.ShortError(err),
 	)
 
-	log.L().Error("failed to covert kv value", logutil.RedactAny("origVal", original.GetValue()),
+	logger.Error("failed to covert kv value", logutil.RedactAny("origVal", original.GetValue()),
```

Comment:
```suggestion
	logger.Error("failed to convert kv value", logutil.RedactAny("origVal", original.GetValue()),
```

Seems a typo. Would you mind fixing it in this PR?

Reply:
done

Reply:
PTAL @buchuitoudegou